### PR TITLE
Update conditional.md

### PR DESCRIPTION
**Pull request by JamesH65**
Updates the conditonal model filters section to use a table rather than long winded description. Should be easier to keep updated this way.

### CODE REVIEW

**Comment by JamesH65** [comment]
Yes, the Pi2 is the Pi2, irrespective of the SoC on it.

File: configuration/config-txt/conditional.md [hunk 1]
```diff
@@ -14,17 +14,18 @@ This is the most basic filter. It resets all previously set filters and allows a
...
-Any settings below a `[pi2]` filter will only be applied to Pi 2 hardware. The `[pi3]` filter is applicable to Pi 3 and Compute Module 3 hardware, while `[pi0]` covers Pi Zero and Pi Zero W.
+The conditional model filters are applied according to the following table.
 
-    [pi1]
-    [pi2]
-    [pi3]
-    [pi3+]
-    [pi0]
-    [pi0w]
+| Filter | Applicable Model |
```
[the remark this answers is not in the document]
Comment (JamesH65) [reply configuration/config-txt/conditional.md]: will change.

File: configuration/config-txt/conditional.md [hunk 2]
```diff
@@ -14,17 +14,18 @@ This is the most basic filter. It resets all previously set filters and allows a
...
-    [pi0]
-    [pi0w]
+| Filter | Applicable Model(s) |
+|--------|------------------|
+| [pi1] | Model A, Model B, Compute Module |
+| [pi2] | Model 2B (2836 or 2837 based) |
+| [pi3] | Model 3B, Compute Module 3 |
+| [pi3+]| Model 3A+, Model 3B+ |
+| [pi0] | Zero |
+| [pi0w]| Zero W, Zero WH |
```
Comment (lurch) [inline configuration/config-txt/conditional.md]: I've just spotted the section below which says
> It is important to note that the Raspberry Pi Zero W will see the contents of [pi0w] AND [pi0]. Likewise, a Raspberry Pi 3B Plus sees [pi3+] AND [pi3].

I wonder if that means the table above ought to say:
```
| [pi3] | Model 3B, Compute Module 3, Model 3A+, Model 3B+ |
| [pi3+]| Model 3A+, Model 3B+ |
| [pi0] | Zero, Zero W, Zero WH |
| [pi0w]| Zero W, Zero WH |
```
?

Comment (pelwell) [reply configuration/config-txt/conditional.md]: Yes - that looks correct.

Comment (JamesH65) [reply configuration/config-txt/conditional.md]: OK, will update as per Andrew's suggestion.

**Pull request merged**